src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -1299,9 +1299,16 @@ def create_file_format_statement(
 
 
 def infer_schema_statement(
-    path: str, file_format_name: str, options: Optional[Dict[str, str]] = None
+    path: str,
+    file_format_name: str,
+    options: Optional[Dict[str, str]] = None,
+    ordered: bool = True,
```

Comment:
I understand the intention is to provide flexibility but looking at the existing callers, I don't think there is a way for users to pass the value through any public api.

based upon the test result, this seems to be a pretty safe change